storybook/src/components/Blockquote/Blockquote.stories.tsx

```diff
@@ -21,31 +21,23 @@ const meta = {
       table: { disable: false },
     },
   },
-} satisfies Meta<typeof Blockquote>
-
-export default meta
-
-type Story = StoryObj<typeof meta>
-
-export const Default: Story = {
   decorators: [
```

Comment:
Nice fix! We discussed extracting this decorator into a reusable one, is that possible? Maybe add a `common` folder next to `docs` and `components`

Reply:
Yes, added!